src/native/libs/System.Security.Cryptography.Native/opensslshim.c

```diff
@@ -116,8 +116,21 @@ static void OpenLibraryOnce(void)
         DlOpen(MAKELIB("10"));
     }
 
-    // FreeBSD uses a different suffix numbering convention.
-    // Current supported FreeBSD releases should use the order .11 -> .111
+    // FreeBSD uses a different suffix numbering convention
+    // The ports version of OpenSSL is used over base where possible
+    if (libssl == NULL)
+    {
+        // FreeBSD uses .12 for OpenSSL 3.0 from ports
```

Comment:
Should this be under FreeBSD ifdef? It looks very FreeBSD specific, it is unlikely to help on other OSes.